website/src/hoc/firebase/WithFirebaseSimple.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { PureComponent } from 'react';
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
+function WithFirebaseSimple(App) {
+  return class extends PureComponent {
+    render() {
+      return (
+        <App
+          {...this.props}
+          firebase={firebase}
+        />
+      );
+    }
+  };
+}
+
+export default WithFirebaseSimple;
```

Comment:
I thought we decided as a group to not use `WithFirebaseSimple`. @LiuYuHao has gotten tests to work with `firebase` without using the exports here I think.

@LiuYuHao  please comment.